### DISCUSSION TITLE
Setup push notifications with next-pwa

### DISCUSSION THREAD

**Discussion by vamsimudadla** [shadowwalker/next-pwa#462] [Q&A]
Is next-pwa enough for setting up push notifications or do we need any third party services like onesignal to send and manage notifications. If next-pwa is enough to setup, please tell me how to do it to make it work both in online and offline mode.

**Comment by ImBIOS** [comment]
@vamsimudadla It seems this repo is dead and the maintainer cannot be contacted for so long now. I've made an immediate response by resurrecting this repo [ImBIOS/next-pwa](https://github.com/ImBIOS/next-pwa).
